apps/system/test/unit/mock_statusbar.js

```diff
@@ -25,6 +28,7 @@ var MockStatusBar = {
   },
 
   updateNotification: function(count) {
+    /* jshint -W053 */
```

Comment:
Jshint is not happy with `new Number(count)`, but I think it's ok for the tests so simply ignore it.